Source/Tx.Network/Udp/BaseUdpReceiver.cs

```diff
@@ -76,114 +111,16 @@ public int AvailableBytes
         {
             get
             {
-                return this.socket.Available;
+                return this.availableSocketBytesGetter();
```

Comment:
That will throw NullRef exception if there are no subscribers, please handle that to provide more meaningful error instead.